pkg/harvester/edit/harvesterhci.io.virtualmachinetemplateversion.vue

```diff
@@ -205,6 +205,11 @@ export default {
       if (tab.name === 'advanced') {
         this.$refs.yamlEditor?.refresh();
       }
+    },
+
+    updateNameNsDescriptionValue(newValue) {
+      this.templateValue.metadata.name = newValue.metadata.name;
+      this.templateValue.metadata.namespace = newValue.metadata.namespace;
```

Comment:
The issue is that line 166 is missing `this.templateValue?.metadata?.name` in create mode.
Fix it by updating templateValue as the user types.